.github/actions/github-pull-request-action/action.yaml

```diff
@@ -0,0 +1,70 @@
+---
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2025 The Linux Foundation
+
+# github-pull-request-action
+name: "#️⃣ GitHub Pull Request"
```

Comment:
Rename the action to "GitHub Pull Request Info", we are'nt creating PR's by only deriving some info about the PR's here.